molecule/requirements.txt

```diff
@@ -3,8 +3,8 @@ ansible==7.0.0
 ansible-compat==3.0.0
 ansible-core==2.14.2
 docker==6.1.3
-molecule==4.0.4
-molecule-docker==2.1.0
+molecule<5
+molecule-docker @ git+https://github.com/ansible-community/molecule-docker@main
```

Comment:
Is there a reason to change the source to straight to github?

Reply:
Yes, there's a fix that went in after 2.1.0, [cb1c1c31a4a468fb243fc2000656504a3ca00bb3](https://github.com/ansible-community/molecule-docker/commit/cb1c1c31a4a468fb243fc2000656504a3ca00bb3)
that fixes a bug where the molecule.yml would get overwritten during image building. (I introduced the simple image-building to get python3 on centos7. But I think it's a good way for us to always stay current on all distribution images, instead of the geerlingguy images)

Reply:
Also, the repo is deprecated/archived, so there won't come any further changes to it, but I could pin it to the hash instead?

We should probably go over to molecule 5, but I'd rather get these refactors in before overhauling the new molecule version.